src/parsers/standardfiles_parser.jl

```diff
@@ -14,18 +14,6 @@ sys = parse_standard_files(
 
 """
 function parse_standard_files(file::String; kwargs...)
```

Comment:
I'm not sure that this function is helpful.  It provides a second way of doing the same thing but doesn't add value (other than being a single step).

Reply:
We can remove this function and leave Systems as the only entry point for Matpower data.